pr.codecept.js

```diff
@@ -25,6 +25,13 @@ exports.config = {
         ],
       },
     },
+    MongoDB: {
+      require: './tests/helper/mongoDB.js',
+      host: '127.0.0.1',
+      port: 27017,
+      username: 'root',
+      password: 'root-password',
```

Comment:
please change this according to the compose mongo details from my PR

Reply:
done, thanks